package.json

```
@@ -20,7 +21,10 @@
     "test": "react-scripts test",
     "eject": "react-scripts eject",
     "storybook": "storybook dev -p 6006",
-    "build-storybook": "storybook build"
+    "build-storybook": "storybook build",
+    "ndev": "next dev",
```

It's better if you replace existing scripts with running dev mode:
```suggestion
    "dev": "next dev",
```

for build and start either